airgun/entities/contentview_new.py

```diff
@@ -63,6 +63,21 @@ def publish(self, entity_name, values=None, promote=False, lce=None):
         view.wait_displayed()
         return view.versions.table.read()
 
+    def check_publish_banner(self, cv_name):
+        """Check if the needs_publish banner is displayed on the content view index page"""
+        view = self.navigate_to(self, 'All')
+        self.browser.plugin.ensure_page_safe(timeout='5s')
+        view.wait_displayed()
+        if not view.table.is_displayed:
+            # no table present, no CVs in this Org
+            return None
+        view.search(cv_name)
+        view.table[0][6].widget.item_select('Publish')
+        view = ContentViewVersionPublishView(self.browser)
```

Comment:
```suggestion
        view = self.navigate_to(self, 'Publish', entity_name=cv_name)
```
@sambible could you try this and see if it does the same thing as you want to do here?